### PR TITLE
Added Kruskals algorithm

### PR DESCRIPTION
**Pull request by Bhuvan506**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: graphs/Kruskals.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Kruskal's algorithm in Python
+
```
Comment (ZeroDayOwl) [inline graphs/Kruskals.py:2]: Add some more description and if possible, a credible source.

File: graphs/Kruskals.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+class Graph:
+    def __init__(self, vertices):
+        self.V = vertices
+        self.graph = []
```
Comment (ZeroDayOwl) [inline graphs/Kruskals.py:7]: Requires doctests for each of the function you add.

File: graphs/Kruskals.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    def add_edge(self, u, v, w):
```
Comment (ZeroDayOwl) [inline graphs/Kruskals.py:9]: Don't use single letter variable.

File: graphs/Kruskals.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    #  Applying Kruskal algorithm
+    def kruskal_algo(self):
+        result = []
+        i, e = 0, 0
+        self.graph = sorted(self.graph, key=lambda item: item[2])
+        parent = []
+        rank = []
+        for node in range(self.V):
+            parent.append(node)
+            rank.append(0)
```
Comment (ZeroDayOwl) [inline graphs/Kruskals.py:39]: Try to write comments explaining how the function works. Check other algorithms in the repo for reference.

**Review by ZeroDayOwl — CHANGES_REQUESTED** [review]

**Comment by cclauss** [comment]
Duplicate https://github.com/TheAlgorithms/Python/search?q=Kruskal&type=code

**Pull request closed without merging**